XenAdmin/Actions/GUIActions/DownloadUpdatesXmlAction.cs

```diff
@@ -293,6 +294,12 @@ protected virtual XmlDocument FetchCheckForUpdatesXml(string location)
             var xdoc = new XmlDocument();
             var uri = new Uri(location);
             var proxy = XenAdminConfigManager.Provider.GetProxyFromSettings(Connection, false);
+            
+            var uniqueIdHash = GetUniqueIdHash();
+
+            string buildText = Helpers.CommonCriteriaCertificationRelease
+                        ? string.Format("{0}: {1}", Program.Version.Revision, Messages.COMMON_CRITERIA_TEXT)

```

Comment:
We don't have any CC versions any more; but even if we did, I don't think we want the COMMON_CRITERIA_TEXT in the user-agent. Let's just lose it.